lmfdb/elliptic_curves/elliptic_curve.py

```diff
@@ -717,20 +720,40 @@ def __init__(self):
             label="Semistable",
             example="Yes",
             knowl="ec.semistable")
+        cm_opts = []
+        for t in cm_j_invariants_and_orders(QQ):
+            el = (str(t[0]*t[1]**2), t[0]*t[1]**2)
+            if not el in cm_opts:
+                cm_opts.append(el)
+        cm_opts = sorted(cm_opts, key =lambda el: el[1], reverse = True)
+        cm_opts.insert(0, ("",""))
```

Comment:
You should put this in the array to begin with (where you set `cm_opts=[]`), that way you don't need an extra line to insert it.

Reply:
If I start with `cm_opts = [('','')]` then the sorted doesn't work.

Reply:
In any case, I'll just hard code the values and avoid this entirely.